pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -44,9 +44,8 @@ def load_earth_free_air_anomaly(resolution="01d", region=None, registration=None
 
     registration : str
         Grid registration type. Either ``"pixel"`` for pixel registration or
-        ``"gridline"`` for gridline registration. Default is ``None``, where
-        a pixel-registered grid is returned unless only the
-        gridline-registered grid is available.
+        ``"gridline"`` for gridline registration. Default is ``"gridline"``
+        when available.
```

Comment:
What about rewording it explicitly to something like this? Ref: https://www.generic-mapping-tools.org/remote-datasets/earth-faa.html

```suggestion
        ``"gridline"`` for gridline registration. Default is ``"gridline"``
        for all resolutions except ``"01m"`` which is ``"pixel"`` only.
```